composables/types.ts

```diff
@@ -216,7 +216,9 @@ export type Base = {
   title: string
   website?: string
   showLatestAdditions?: boolean
+  latestAdditions: number[]
```

Comment:
```suggestion
  latestAdditions?: number[]
```
y'a qqs endroits où on crée une base vide et du coup c'est relou avec les champs obligatoires

Reply:
mince un mauvais rebase je l'avais ajouté, j'ai du le supprimer par mégarde. Merci de ton attention